project_generator/templates/makefile_gcc.tmpl

```diff
@@ -40,7 +40,7 @@ TARGET_EXT =
 TARGET_EXT = a
 {% endif %}
 
-LD_SCRIPT = {{linker_file}}
+LD_SCRIPT = {% for file in linker_file %} {{file}} {% endfor %}
```

Comment:
Yes. More than one linker script is possible. I don't know a use case for it, so it might be best to throw a warning anyway.